tests/test_cadquery.py

```diff
@@ -969,25 +969,42 @@ def testPolarArray(self):
         s = Workplane("XY").polarArray(radius, 0, 180, 6)
         self.assertEqual(6, s.size())
 
+        to_x = lambda l: l.wrapped.Transformation().TranslationPart().X()
+        to_y = lambda l: l.wrapped.Transformation().TranslationPart().Y()
+        to_angle = (
+            lambda l: l.wrapped.Transformation().GetRotation().GetRotationAngle()
+            * 180.0
+            / math.pi
+        )
+
         # Test for proper placement when fill == True
         s = Workplane("XY").polarArray(radius, 0, 180, 3)
-        self.assertAlmostEqual(0, s.objects[1].x)
-        self.assertAlmostEqual(radius, s.objects[1].y)
+        self.assertAlmostEqual(0, to_y(s.objects[1]))
```

Comment:
`x` and `y` seem to be swapped from what they were in all these tests. Is that intended?

Reply:
Related to the comment above